homeassistant/components/logentries.py

```diff
@@ -7,29 +7,31 @@
 import json
 import logging
 import requests
-import homeassistant.util as util
-from homeassistant.const import EVENT_STATE_CHANGED
+
+import voluptuous as vol
+
+from homeassistant.const import (CONF_TOKEN, EVENT_STATE_CHANGED)
 from homeassistant.helpers import state as state_helper
-from homeassistant.helpers import validate_config
+import homeassistant.helpers.config_validation as cv
 
 _LOGGER = logging.getLogger(__name__)
 
-DOMAIN = "logentries"
-DEPENDENCIES = []
+DOMAIN = 'logentries'
 
 DEFAULT_HOST = 'https://webhook.logentries.com/noformat/logs/'
 
-CONF_TOKEN = 'token'
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_TOKEN): cv.string,
+    }),
```

Comment:
I didn't add `PREVENT_EXTRA` because it seems to be the [default](https://github.com/alecthomas/voluptuous/blob/master/voluptuous/schema_builder.py#L176).